docs/policy-only.md

```diff
@@ -0,0 +1,169 @@
+
+Specifically a re-entrance traffic may be traced as follows:<br>
+Denote VIP as cluster IP of a service, SP_IP/DP_IP as respective client and server Pod IP; 
+VPort as service port of a service, TPort as target port of server Pod, and SPort as original 
+source port. The service request's 5-tuples upon first and second entrance to the host network, and
+its reply's 5-tuples would be like
+
+```
+request/service:   
+-- Entering Host Network(via gw0):     SP_IP/SPort->VIP/VPort 
+-- After LB(DNAT):                     SP_IP/SPort->DP_IP/TPort
+-- After Route(to gw0):                SP_IP/SPort->DP_IP/TPort
+
+request/forwrding: 
+-- Entering Host Network(via gw0):     SP_IP/SPort->DP_IP/TPort
+-- After route(to uplink):             SP_IP/SPort->DP_IP/TPort
+
+reply:
+-- Entering Host Network(via uplink):  DP_IP/TPort -> SP_IP/SPort
+-- After LB(DNAT):                     VIP/VPort->SP_IP/Sport
+-- After route(to gw0):                VIP/VPort->SP_IP/Sport
+```
+
+#### Routing 
+Note that the request with destination IP DP_IP needs to be routed differently in LB and 
+forwarding cases.(This differs from encap traffic where all traffic flows including post LB
+service traffic share the same ``main`` route table.) Antrea creates a customized
+``antrea_service`` route table, it is used in
+conjunction with ip-rule and ip-tables to handle service traffic. Together they work as follows
+1. At Antrea initialization, an ip-tables rule is created in ``mangle table`` that marks IP packets
+with service IP as destination IP and are from gw0. 
+1. At Antrea initialization, an ip-rule is added to select ``antrea_service`` route table as routing
+table if traffic is marked in 1).
+1. At Antrea initialization, a default route entry is added to ``antrea_service`` route table to
+forward all traffic to ``gw0``.
+
+The outcome may be something like this
+```bash
+ip neigh | grep gw0
+169.254.253.1 dev gw0 lladdr 12:34:56:78:9a:bc PERMANENT
+
+ip route show table 300 #tbl_idx=300 is antrea_service
+default via 169.254.253.1 dev gw0 onlink 
+
+ip rule | grep gw0
+300:	from all fwmark 0x800/0x800 iif gw0 lookup 300 
+
+iptables -t mangle  -L ANTREA-MANGLE 
+Chain ANTREA-MANGLE (1 references)
+target     prot opt source               destination         
+MARK       all  --  anywhere             10.0.0.0/16          /* Antrea: mark service traffic */ MARK or 0x800
+MARK       all  --  anywhere            !10.0.0.0/16          /* Antrea: unmark post LB service traffic */ MARK and 0x0
+```
+
+The above configuration allows Pod-to-Service traffic to use ``antrea_service`` route table after
+load balancing, and to be steered back to OVS bridge for Pod NetworkPolicy processing.
+
+#### Conntrack
+Note also that with re-entrance traffic a service request, after being load balanced and routed
+back to OVS bridge via ``gw0``, has exactly the same 5-tuple as when it re-enters the host network
+for forwarding.
```

Comment:
consider rephrasing this first paragraph here..

Reply:
done.